test/test_transforms.py

```diff
@@ -428,6 +428,13 @@ def test_to_tensor(self):
         height, width = 4, 4
         trans = transforms.ToTensor()
 
+        with self.assertRaises(TypeError):
+            trans(np.array([1, height, width]).tolist())
```

Comment:
this creates a 1d tensor containing the values `[1, height, width]`, is that what you meant?
Maybe you wanted something like
```python
np.random.rand(1, height, width)
```
to generate a 3d array?

Reply:
Yes. My bad. Sorry for making a stupid mistake (｡•́︿•̀｡).